node/docs/minagent.md

```diff
@@ -30,6 +30,8 @@ Use the details below to determine when specific agent features were added:
   - Added in 2.144.0. Used node v10.x
 * `node16` handler
   - Added in 2.206.1. Used node v16.x
+* `node20` handler
+  - Added in 3.225.0. Used node v20.x
```

Comment:
We supported node20 handler in this [PR](https://github.com/microsoft/azure-pipelines-agent/pull/4346), and it's published in release [3.224.0](https://github.com/microsoft/azure-pipelines-agent/releases), so my understanding is that here we will require minimum agent version to be >= 3.224.0 to enable node20 support.